multiterm/src/Makefile.am

```diff
@@ -24,6 +24,8 @@ multiterm_la_SOURCES = \
 	terminal.vala
 
 multiterm_la_CFLAGS = \
+	-DGETTEXT_PACKAGE='"$(GETTEXT_PACKAGE)"' \
+	-DLOCALEDIR='"$(localedir)"' \
```

Comment:
Hum, actually I just tried and it's not possible for `LOCALEDIR` as it has to be changeable when running Make itself, so it has to be passed through there.
So we could have this for `GETTEXT_PACKAGE`, but I doubt it's very sensible to use it only for this.